src/ocean/io/Path.d

```diff
@@ -302,9 +302,7 @@ package struct FS
 
                 static bool isWritable (cstring name)
                 {
-                        stat_t stats = void;
-
-                        return (getInfo(name, stats) & O_RDONLY) is 0;
+                        return posix.access(name.ptr, W_OK) == 0;
```

Comment:
is name guaranteed to be 0-terminated here?

Reply:
Yes, as in all these methods in the same `package` struct.